deploy/cdk_exec_policy/cdkExecPolicy.yaml

```diff
@@ -196,9 +206,7 @@ Resources:
             Effect: Allow
             Action:
               - 'cloudformation:*'
-            Resource:
```

Comment:
Why did you change this?

Reply:
@zsaltys , this  was a sync issue on my end, thank you for notifying. I reverted this to previous permission set now.